docs/articles/extending/multihost-hosts.rst

```diff
@@ -0,0 +1,129 @@
+Multihost Hosts
+###############
+
+:class:`~pytest_mh.MultihostHost` has access to the host part of the
+configuration file. Its main purpose is to setup and teardown the host and
+prepare it to run the tests and provide access to the main connection to the
+host over which is it possible to run commands on the host (see
+:attr:`~pytest_mh.MultihostHost.conn` attribute).
```

Comment:
Something doesn't sound correct in this sentence, but I don't know what it is. @justin-stephenson do you mind taking a look?

Reply:
In my opinion, something like this would be better.

`Its main purpose is to setup and teardown the host, preparing it to run tests. It also provides the main connection to the host making it possible to run remote commands.`

Reply:
Ok, thank you Justin! I'm adding that to my corrections.